File: js/src/maplestory/api/dto/guild/guildDto.ts
import { GuildBody } from '../../response/guild/guildBody';

/**
 * 길드 식별자 정보
 */
class GuildDto {
  /**
   * 길드 식별자
   */
  oguildId: string;

  constructor(obj: GuildBody) {
    const { oguild_id } = obj;

    this.oguildId = oguild_id;
  }
}

export { GuildDto };

File: java/src/main/java/dev/spiralmoon/maplestory/api/msea/dto/guild/GuildDTO.java
package dev.spiralmoon.maplestory.api.msea.dto.guild;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 * Guild identifier information
 */
@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class GuildDTO implements dev.spiralmoon.maplestory.api.common.dto.guild.GuildDTO {

    /**
     * Guild identifier
     */
    @JsonProperty("oguild_id")
    private String oGuildId;
}

